### DISCUSSION TITLE
I can't upload a proof of my academic status to request a discount

### DISCUSSION THREAD

**Discussion by Promenak** [community/community#20103] [GitHub Education] [answered]
It says to upload the proof but I don't see any upload button and there's just a Take a picture button which uses my laptop's camera to take a picture but I have the proof as a .jpg file on my laptop and am willing to upload that file

**Comment by ghost** [comment]
@Un-known-User 
 how this! , the button exists you just make reload to the page and  make sure u use  edu email 
 note: i just make it NOw

**Reply by Zoltus** [reply]
> It says to upload the proof but I don't see any upload button and there's just a Take a picture button which uses my laptop's camera to take a picture but I have the proof as a .jpg file on my laptop and am willing to upload that file

Its in the next step, after you add your school issued email.
![upload-proof-status](https://user-images.githubusercontent.com/23402980/177575073-1e1d6d71-a968-4eed-ae59-7ced0185878d.png)
![upload-image-link](https://user-images.githubusercontent.com/23402980/177575068-2d2ba7fd-8247-4a54-90ae-0ff1f8a8192e.png)

I hope u get it!

**Reply by Promenak** [reply]
> > It says to upload the proof but I don't see any upload button and there's just a Take a picture button which uses my laptop's camera to take a picture but I have the proof as a .jpg file on my laptop and am willing to upload that file
> 
> Its in the next step, after you add your school issued email. ![upload-proof-status](https://user-images.githubusercontent.com/23402980/177575073-1e1d6d71-a968-4eed-ae59-7ced0185878d.png) ![upload-image-link](https://user-images.githubusercontent.com/23402980/177575068-2d2ba7fd-8247-4a54-90ae-0ff1f8a8192e.png)
> 
> I hope u get it!
I don't know why but that button is not in my page
![Screenshot (4)](https://user-images.githubusercontent.com/88489031/177576571-691e7aa9-2d37-4d9e-9e62-f5ba120d04c6.png)

**Reply by ghost** [reply]
@Un-known-User 
  are U adding educational email to your account?

**Reply by Promenak** [reply]
> @Un-known-User are U adding educational email to your account?

Nope I'm using the same email I created my account with

**Reply by Zoltus** [reply]
> 

You have to add school issued email which is verified by github. It wont change your profile email.

**Reply by ghost** [reply]
@Un-known-User 
that is a problem  please go to email and add your educational email and try again
got to email setting and add it  
![image](https://user-images.githubusercontent.com/108471229/177579482-e41a2727-3919-4d09-bfc7-df9cb12c42a6.png)

![image](https://user-images.githubusercontent.com/108471229/177579583-9e0150dc-e0fe-4cff-90ca-a5d10d2aa205.png)

**Reply by Promenak** [reply]
> @Un-known-User that is a problem please go to email and add your educational email and try again got to email setting and add it 

I remember sending a request with this email last week without any problems and I could upload an image back then but my request was rejected because the image I provided wasn't enough to prove my academic status so I wanted to apply another time but this time this thing happened and I couldn't find the upload button so I can only apply once with every email?

**Reply by Zoltus** [reply]
> > @Un-known-User that is a problem please go to email and add your educational email and try again got to email setting and add it
> 
> I remember sending a request with this email last week without any problems and I could upload an image back then but my request was rejected because the image I provided wasn't enough to prove my academic status so I wanted to apply another time but this time this thing happened and I couldn't find the upload button so I can only apply once with every email?

I think you should contact the support

**Reply by ghost** [reply]
I think so, but u can  try using the school email  or any email to make sure  your thought is right or not

**Reply by Promenak** [reply]
> I think you should contact the support

yup apparently that's the only thing I can do

**Reply by Promenak** [reply]
> I think so, but u can try using the school email or any email to make sure your thought is right or not

I used this email in school too

**Comment by ghost** [comment, marked as answer]
@Un-known-User 
look  to this https://github.com/orgs/github-community/discussions/17736
 Elise from Github said  u can use a camera phone  : )

**Reply by Promenak** [reply]
> @Un-known-User look to this https://github.com/orgs/github-community/discussions/17736 Elise from Github said u can use a camera phone : )

Ah thanks
I thought this was some sort of bug or issue with the site
I'll try to capture a nice pic of my proof and apply this time :D

**Reply by ghost** [reply]
Good luck to you <3
